engine/execution/computation/computer/computer.go

```diff
@@ -481,9 +491,10 @@ func (e *blockComputer) executeTransaction(
 	e.metrics.ExecutionTransactionExecuted(
 		time.Since(startedAt),
 		tx.ComputationUsed,
-		memAllocAfter-memAllocBefore,
 		tx.MemoryEstimate,
+		memAllocAfter-memAllocBefore,
```

Comment:
maybe we should just get rid of per txn memory usage estimate base on mem alloc.

it doesn't seem very meaningful since memory usage is dominated by background task (snapshotting, lib p2p, etc.) and semi-permanent allocation (trie / storage).  The number will become even less meaningful when we start parallelizing some aspects of transaction (pre)processing.

Reply:
@dsainati1 thoughts on this? do we still need it?

Reply:
Sorry I think I missed this comment originally, but Cadence needs some way to sanity check our memory metering estimates to make sure changes we make or features we add (especially in light of the upcoming Stable Cadence changes) don't underestimate memory usage. So if we get rid of this ideally we'd replace it with a better estimate.

Reply:
got it lets leave it the way it is in this PR and we could address it later.